apps/console/src/features/claims/components/edit/external-dialect/edit-dialect-details.tsx

```diff
@@ -108,9 +108,17 @@ export const EditDialectDetails: FunctionComponent<EditDialectDetailsPropsInterf
                             data-testid={ `${ testId }-form-dialect-uri-input` }
                         />
                         <Divider hidden />
```

Comment:
Shall we move this just above the `PrimaryButton` to avoid extra spacing?

Reply:
**Done, find the below image**

![Screenshot from 2021-02-01 11-57-50](https://user-images.githubusercontent.com/33889049/106425289-b54dbd00-6489-11eb-8f2d-f5e09bea7c6f.png)

Reply:
Nice 👍